src/python/ensembl/io/genomio/database/meta_getter.py

```diff
@@ -50,59 +50,50 @@ def get_meta_values(server_url: URL, db_name: str, input_queries: StrPath) -> di
     meta_populated = False
 
     # Check input type and populated query list
-    if isinstance(input_queries, PosixPath):
-        with Path(input_queries).open(mode="r", encoding="UTF-8") as fh:
+    if isinstance(meta_keys, PosixPath):
+        with Path(meta_keys).open(mode="r", encoding="UTF-8") as fh:
             for line in fh.readlines():
                 meta_key = line.strip()
                 query_meta_keys.append(meta_key)
-    elif isinstance(input_queries, list):
-        query_meta_keys = input_queries
+    elif isinstance(meta_keys, list):
+        query_meta_keys = meta_keys
```

Comment:
Apologies about this change, should have detected it sooner: treat it as a file path if it is not a list (to cover all the possible types a file path can be given in)
```suggestion
    if isinstance(meta_keys, list):
        query_meta_keys = meta_keys
    else:
        with Path(meta_keys).open(mode="r", encoding="UTF-8") as fh:
            for line in fh.readlines():
                meta_key = line.strip()
                query_meta_keys.append(meta_key)
```